### ISSUE TITLE
New Commented Topics 

### ISSUE THREAD

**Issue by maduranga001** [ActiveForums/ActiveForums#225]
Hi,

Please can you help me to get all topics just after commented to a topic from a user? I need only to show the first 10 topics has updated comments. Currently "AF What's New" module has this. But it is giving the comments. If two users commented to same topic, then are two items with same topic in the list. I need to show only one with the latest topics.

If you have an idea how to edit the SQL for What's New module or if you can help me to get the list in an SQL that is also fine. SO we can use it separatly without doing any changes to the exisiting modules and SQLs.

I need the top 10 updated topics with commenets without repeating items, with Author,Commenter, Post posted date, Comment Posted date and the Forum category anme and the URL.

Pleas can you help me for this?

Appreciate the support.

Thank You!

**Comment by maduranga001** [comment]
I really appreciate if you can give me an SQL to load recent commented topics and not repeaded topics items. with the last commented user and the original user.

Please.

Thank You!

**Comment by artfuld** [comment]
Not sure what you are asking for but I think that the view ForumLastPosts is what you are looking for.
vw_activeforums_ForumLastPosts

**Comment by maduranga001** [comment]
Hi,

Thank you very much for sending me an answer.

Sorry If I did not mentioned it properly. I checked the view you mentioned above.

But I need this.

If somebody commented in a post, It should come first. This is same as the "Active topics" section in the forum module. But How can I load this view in the home page? The user need to go to the "Forum" page and need to click on the "Active Topics". Can i give the same view as it is in the home page?

I was trying to get the same with using "What's New" Module instead of above. But it is giving the same topic if 3 users commented for a same topic repeatedly.

Did I mentioned this properly now? Did you get an idea about my needs? Sorry if i could not mentioned it well.

Please help me. The users are requesting this too many times. It is a free site too. But lot of users are active in this forum site now.

Hope you cn help me for my trouble.

Thank You!

**Comment by maduranga001** [comment]
www.website.com/Forum/afv/grid/afgt/activetopics/ts/80640   is the ideal view.
At-least this - www.website.com/Forum/afv/grid/afgt/activetopics 

Please can you help me? I need to load this view when load the home page.

Thank You!

**Comment by artfuld** [comment]
So, if I understand you correct you want the what's new-module to only show one post per topic?

**Comment by maduranga001** [comment]
Yes Exactly Joakim. Please can you help me? I am fine if you can tell me where I nee to do any changes in the database too. Not an expert. Bu can follow your instructions. Thank You very very much for your supporting.

**Comment by artfuld** [comment]
I think that this feature is something that more people can benefit from and we should try to implement it to the module directly.

**Comment by maduranga001** [comment]
Ok Joakim. Can we modify the current SQL and can we use this feature as to has now? Just avoiding the same topic loading? If yes, what is the SQL and how can i modify it? I know this will affect to the next upgrading. But I can change this again after the next module upgrading if you can help me to modify the SQL.

Most of forum sites have ths feature and this is why users are asking for this.

Thank You!

**Comment by supermansrf** [comment]
@artfuld I completely agree.  I think that making some changes to the whats new module to allow things like this, or top posters, or top posts, having multiple instances available anywhere on a site would be a great idea.  I started looking into this a while back and it just lost steam.  

The whats new module hasn't been looked at as long as I can remember.  I started using it and quickly realized that it needs help.  I can see where its potential long term would be very helpful.  Unfortunately it isn't there yet.  

Anyone who would like to take this on by all means go for it.  I would love to see the whats new module offer additional options, tokens, whatever.  It probably needs skinned to match the new UI too at some point.

**Comment by maduranga001** [comment]
Hi Seth,

Thank you for your reply. We all can be happy if we have this module with new features. Can we expect this in near future? Between is there no luck to use the current module with some changes for my needs? I mean like in SQL? as this is the main view most of forums has now?

Thank You very much for your replies.
